configs/encapsulate_in_connect.v3.yaml

```diff
@@ -0,0 +1,38 @@
+admin:
```

Comment:
do we have docs explaining how this feature works as well?

Reply:
Just what's in this PR.  I think folks will mainly only use this for TCP tunneling - while there is a desire to have Envoy generate CONNECT requests AFIK it's only for HTTP/1.1, where for HTTP/2 the only planned use is tunneling TCP (by Google and folks who asked for https://github.com/envoyproxy/envoy/issues/1630)